app/src/test/java/com/github/se/orator/model/profile/UserProfileViewModelTest.kt

```diff
@@ -188,4 +188,68 @@ class UserProfileViewModelTest {
     val selectedFriend = viewModel.selectedFriend.first()
     Assert.assertEquals(friendProfile, selectedFriend)
   }
+
```

Comment:
The test cases check friend list size before and after modification, which is great for verifying list integrity. Consider adding some documentation and comment for readers to understand better the test cases, if we ever need to modify them.